bin/sdk/gutenberg.js

```diff
@@ -10,14 +12,65 @@ exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 		cssFilename: '[name].css',
 		externalizeWordPressPackages: true,
 	} );
-	const editorScript = path.join( inputDir, 'editor.js' );
-	const viewScript = path.join( inputDir, 'view.js' );
+
+	const presetPath = path.join( inputDir, 'index.json' );
+
+	let editorScript;
+	let viewBlocksScripts;
+	let viewScriptEntry;
+	if ( fs.existsSync( presetPath ) ) {
+		const DIRECETORY_DEPTH = '../../';
+		const presetBlocks = require( presetPath );
+		// Find all the shared scripts
+		const sharedPath = path.join( inputDir, 'shared' );
+		const sharedUtilsScripts = fs
```

Comment:
We seem to duplicate code for similar operations here - we could definitely pluck out some of it in helper functions.